business_requirement_etherpad/views/business_view.xml

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
-        <record id="view_business_requirement_form" model="ir.ui.view">
-                <field name="name">business.requirement.form</field>
-                <field name="model">business.requirement</field>
-                <field name="inherit_id" ref="business_requirement.view_business_requirement_form"/>
-                <field name="arch" type="xml">
-                    <xpath expr='//page[@string="Logs"]' position='before'>
-                        <page string="Notes">
-                            <group>
-                                <field name="notes" widget="pad"/>
-                            </group>
-                        </page>
-                    </xpath>
-                </field>
-        </record>
-    </data>
-</openerp>
+<odoo>
+    <record id="view_business_requirement_form" model="ir.ui.view">
+            <field name="name">business.requirement.form</field>
+            <field name="model">business.requirement</field>
+            <field name="inherit_id" ref="business_requirement.view_business_requirement_form"/>
+            <field name="arch" type="xml">
+                <xpath expr='//page[1]' position='after'>
```

Comment:
- [ ] use `"` as attribute separator
- [ ] why using absolute index instead of the tab name?